.eslintrc

```diff
@@ -63,7 +63,9 @@
     }],
     "lines-between-class-members": "off",
     "max-classes-per-file": "off",
-    "spaced-comment": "warn"
+    "spaced-comment": "warn",
+    "import/no-cycle": "off",
+    "import/no-named-as-default": "off"
```

Comment:
When introducing https://github.com/oviohub/prism-frontend/pull/92/commits/d5fad7fdce33047e82747c4ce2843c7ef0c3f90a to fix linting issues in Windows, it introduced these errors at various places in the codebase. I turned them off for now, is it a concern?

Reply:
It will get solved in #90 so let's not touch this file in this PR

Reply:
Got it, I'll revert the commit